### PR TITLE
Add unique identifiers for iCal exports, to allow Google Calendar imports without User Auth

### PR DESCRIPTION
**Pull request by NotANormalNerd**
Implements #56

### CODE REVIEW

File: letsmeet/events/feeds.py [hunk 1]
```diff
@@ -84,3 +86,20 @@ def items(self, obj):
         if hasattr(obj, 'userprofile'):
             return obj.userprofile.get_upcoming_yes_events().order_by('-created')
         return []
+
+
+class ICalPersonalUserEventsFeed(ICalCommunityEventsFeed):
```
Comment (michael-k) [inline letsmeet/events/feeds.py]: DRY:
```python
class ICalPersonalUserEventsFeed(ICalUserEventsFeed):
```

File: letsmeet/events/urls.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
     EventDetailView,
     EventRSVPView,
 )
-from .feeds import LatestEventsFeed, ICalCommunityEventsFeed, ICalUserEventsFeed
+from .feeds import LatestEventsFeed, ICalCommunityEventsFeed, ICalUserEventsFeed, ICalPersonalUserEventsFeed
```
Comment (michael-k) [inline letsmeet/events/urls.py]: this line is getting to long

File: letsmeet/users/templates/users/dashboard.html [hunk 3]
```diff
@@ -41,6 +41,7 @@
         <div class="block-area">
           <div class="well tile">
 	      <a class="btn btn-default" href="{% url 'personal_events_ical_feed' %}"><i class="fa fa-calendar"></i> your personal ical calendar</a>
+          <a class="btn btn-default" href="{% url 'personal_events_ical_feed_uuid' uuid=userprofile.personal_ical_uuid %}"><i class="fa fa-calendar"></i> ical calendar (Google importable)</a>
```
Comment (michael-k) [inline letsmeet/users/templates/users/dashboard.html]: indentation matters

**Review by michael-k — CHANGES_REQUESTED** [review]

**Comment by asmaps** [comment]
I think it would also be okay to replace the current feed.

**Comment by NotANormalNerd** [comment]
@michael-k Thank you for the review. I will change the stuff mentioned and come back.
@asmaps Yes it is probabl a good idea. I will remove the current feed and replace it with my implementation.
I will do the changes as soon as I have time.

**Comment by asmaps** [comment]
Will review and merge as soon as I have time.
Anymore comments @michael-k ?

File: letsmeet/communities/models.py [hunk 4]
```diff
@@ -47,6 +47,7 @@ class Community(TimeStampedModel):
     is_deleted = models.BooleanField(default=False)
 
     objects = CommunityManager()
+
     default = models.Manager()   # the default manager
```
Comment (michael-k) [inline letsmeet/communities/models.py]: This is due to a bug in pycodestyle.  Should not be changed in this PR.
See issue 617 of https://github.com/PyCQA/pycodestyle

File: letsmeet/users/templates/users/dashboard.html [hunk 5]
```diff
@@ -40,8 +40,8 @@
 	{% if request.user %}
         <div class="block-area">
           <div class="well tile">
-	      <a class="btn btn-default" href="{% url 'personal_events_ical_feed' %}"><i class="fa fa-calendar"></i> your personal ical calendar</a>
-            </div>
+	        <a class="btn btn-default" href="{% url 'personal_events_ical_feed' uuid=userprofile.personal_ical_uuid %}"><i class="fa fa-calendar"></i> Your ical calendar</a>
+          </div>
```
Comment (michael-k) [inline letsmeet/users/templates/users/dashboard.html]: Indentation is broken, but it was broken before (just in a different way ;) ).
Maybe change it to 4 spaces (to be consistent with the rest of the file).

**Review by michael-k — APPROVED** [review]
LGTM

**Comment by michael-k** [comment]
ping @asmaps @mfa

**Pull request merged**